JabRef/jabref
# Add setting: always add "Cited on pages" text to JStyles.

Closes https://github.com/JabRef/jabref/issues/11691
## Draft PR for coding feedback

### TODO (for me):
[] Update user documentation, open documentation PR.
[] Review developer documentation to see if update necessary. (I think it's unlikely).

### To Test:
1. Toggle the Open- / LibreOffice view under `View > OpenOffice/LibreOffice` (or `Alt + 0`)
2. Click on "Select Style" in the OpenOffice menu which appears (assumedly on the left side of the window).
3. Choose any JStyle.
4. In the same area as "Select Style", click on "Settings"
5. **Expected**: The option "Automatically add "Cited on pages..." at beginning of citation" should appear unchecked.
6. Toggle the setting exactly once by clicking.
7. Fully close and restart the application.
8. Repeat steps 1 (if the OpenOffice view doesn't reopen) and 4.
9. **Expected**: The toggled option should still be checked.
10. Optionally repeat steps 6, then 1 and 4 again. The now twice toggled setting should again still be preserved (aka it should be unchecked).

### Screenshot Preview of feature:
![11691-pr-ui-screenshot](https://github.com/user-attachments/assets/aa75c8f1-0820-4e56-89da-f8f5f8b73621)



### Mandatory checks



- [x] Screenshots added in PR description (for UI changes)
